spec/javascripts/components/accessible-autocomplete-spec.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-env jasmine, jquery */
+/* global GOVUK */
+
+describe('An accessible autocomplete component', function () {
+  'use strict'
+
+  function loadAutocompleteComponent () {
+    window.setFixtures(html)
+    var autocomplete = new GOVUK.Modules.AccessibleAutocomplete()
+    autocomplete.start($('.app-c-accessible-autocomplete'))
```

Comment:
This needs updating in line with the changes to the start/init/jQuery/etc. stuff. I think it should be something like:

```JavaScript
var el = document.querySelector('.app-c-accessible-autocomplete')
var autocomplete = new GOVUK.Modules.AccessibleAutocomplete(el)
autocomplete.init()
```

Reply:
Good catch, also need to update the CI so that these tests actually get run 😅

Reply:
I was wondering why there weren't any errors! 😁